image_width: 640
image_height: 480
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [618.92056,   0.     , 317.98132,
           0.     , 619.56126, 249.21714,
           0.     ,   0.     ,   1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.130766, 0.241241, 0.000005, -0.001481, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [611.17255,   0.     , 316.71703,   0.     ,
           0.     , 611.72327, 248.50507,   0.     ,
           0.     ,   0.     ,   1.     ,   0.     ]
